### PR TITLE
add issue templates

### PR DESCRIPTION
**Pull request by kinnou02**
add a precision that data issue have to be handled on the googlegroups
[skip-ci]

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+These issues are only for bugs related to the software navitia, problems on navitia.io related to the data must be addressed on https://groups.google.com/forum/#!forum/navitia
+
+**To Reproduce**
+1. An url of navitia.io where the problem is visible. If the case appear on your navitia instance the datasets that you used.
```
Comment (AurelienLP) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
An url of navitia.io where the problem is visible. If the case appear on your navitia instance the datasets that you used.
```

**Review by AurelienLP — APPROVED** [review]

**Review by benoit-bst — APPROVED** [review]
It will be usefull ! :100:

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+These issues are only for bugs related to the software navitia, problems on navitia.io related to the data must be addressed on https://groups.google.com/forum/#!forum/navitia
```
Comment (nonifier) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
These issues are only for bugs related to the software navitia, problems with navitia.io or related to the data must be addressed on https://groups.google.com/forum/#!forum/navitia
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+**To Reproduce**
+An url of navitia.io where the problem is visible. If the case appear on your navitia instance the datasets that you used.
```
Comment (nonifier) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
Please provide a navitia.io URL that reproduces the problem (you can also refer to our [Playground](http://canaltp.github.io/navitia-playground/play.html)). In the case of a custom Navitia instance, you may provide access the datasets being used.
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
```
Comment (nonifier) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
A clear and concise description of the expected behavior.
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
```
Comment (nonifier) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
If applicable, add screenshots or any resources likely to help us understanding your problem.
```

**Review by nonifier — APPROVED** [review]
Super helpful ! :pray:

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 3 above)
Comment (pbougue) [reply .github/ISSUE_TEMPLATE/bug_report.md]: @nonifier If we mention the playground, we must find a way for people to pay attention that no token is provided in URL.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (pbougue) [reply .github/ISSUE_TEMPLATE/bug_report.md]: @nonifier I'd precise a bit more your suggestion.
```suggestion
These issues are only for bugs related to the software navitia, problems with navitia.io itself (website, access) or related to the data must be addressed on https://groups.google.com/forum/#!forum/navitia```

**Review by pbougue — APPROVED** [review]

**Pull request merged**